app/v2/notifications/post_notifications.py

```diff
@@ -135,6 +140,24 @@ def post_precompiled_letter_notification():
     return jsonify(resp), 201
 
 
+def _seed_bounce_data(epoch_timestamp: int, service_id: str):
+    current_time_ms = int(1000.0 * datetime.now().timestamp())
+    time_difference_ms = current_time_ms - epoch_timestamp
+
+    twenty_four_hours_ms = 86400000
+
+    if 0 <= time_difference_ms <= twenty_four_hours_ms:
+        # We are in the 24 hour window to seed bounce rate data
+        # Check if the seeded data key exists in redis
+        if not redis_store.get(_total_notifications_seeded_key(service_id)):
+            seed_bounce_rate_in_redis.apply_async(service_id)
+            current_app.logger.info("Seeding bounce rate for service_id {}".format(service_id))
+        else:
+            current_app.logger.info("Bounce rate already seeded for service_id {}".format(service_id))
```

Comment:
What if the key was already there with some old data that hasn't expired yet? I'm thinking if we need to run this 2 times in a 24 hour period

Reply:
Not a concern since this will only be used if there is no data in Redis